libs/platform/unit/platform_unit_test.cpp

```diff
@@ -327,36 +327,43 @@ TEST_CASE("epoch_test_stale_items", "[platform]")
     if (ebpf_get_cpu_count() < 2) {
         return;
     }
-
-    auto t1 = [&]() {
-        uintptr_t old_thread_affinity;
-        ebpf_set_current_thread_affinity(1, &old_thread_affinity);
-        ebpf_epoch_enter();
-        void* memory = ebpf_epoch_allocate(10);
-        signal_2.signal();
-        signal_1.wait();
-        ebpf_epoch_free(memory);
-        ebpf_epoch_exit();
-    };
-    auto t2 = [&]() {
-        uintptr_t old_thread_affinity;
-        ebpf_set_current_thread_affinity(2, &old_thread_affinity);
-        signal_2.wait();
-        ebpf_epoch_enter();
-        void* memory = ebpf_epoch_allocate(10);
-        ebpf_epoch_free(memory);
-        ebpf_epoch_exit();
-        signal_1.signal();
-    };
-
-    std::thread thread_1(t1);
-    std::thread thread_2(t2);
-
-    thread_1.join();
-    thread_2.join();
-    std::this_thread::sleep_for(std::chrono::seconds(1));
-    REQUIRE(ebpf_epoch_is_free_list_empty(0));
-    REQUIRE(ebpf_epoch_is_free_list_empty(1));
+    for (size_t i = 0; i < 100; i++) {
```

Comment:
if 100 arbitrary?   maybe
```suggestion
    const size_t test_iterations = 100; 
    for (size_t i = 0; i < test_iterations; i++) {
```

Reply:
I didn't see any response or change for this comment

Reply:
Isn't it part of this commit https://github.com/microsoft/ebpf-for-windows/pull/1332/commits/3ea9b0ae272ba236ce2aadbf2f506a140b6ec270?

Reply:
sort of.  I was looking for some label on the max.  the commit labeled the counter.   If you think it's good enough, feel free to ignore this comment.

Reply:
ok, done.